components/user_list_row_with_error.jsx

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedHTMLMessage} from 'react-intl';
+
+import {Client4} from 'mattermost-redux/client';
+
+import UserStore from 'stores/user_store.jsx';
+
+import * as Utils from 'utils/utils.jsx';
+
+import ProfilePicture from 'components/profile_picture.jsx';
+
+export default class UserListRowWithError extends React.Component {
+
+    static propTypes = {
+        user: PropTypes.object.isRequired,
+        extraInfo: PropTypes.arrayOf(PropTypes.object),
+        actions: PropTypes.arrayOf(PropTypes.func),
+        actionProps: PropTypes.object,
+        actionUserProps: PropTypes.object,
+        userCount: PropTypes.number
+    };
+
+    static defaultProps = {
+        extraInfo: [],
+        actions: [],
+        actionProps: {},
+        actionUserProps: {}
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {};
+
+        this.onError = this.onError.bind(this);
+    }
+
+    onError(errorObj) {
+        this.setState({
+            error: errorObj
+        });
+    }
```

Comment:
This class is identical to `UserListRow` except that it has this function, and renders the error if set